packages/editor-ui/src/stores/assistant.store.ts

```diff
@@ -438,6 +439,30 @@ export const useAssistantStore = defineStore(STORES.ASSISTANT, () => {
 			context: visualContext,
 			question: userMessage,
 		};
+		const payloadSize = getObjectSizeInKB(payload);
```

Comment:
This can throw an error, but does not get caught. Can we handle that better so that UX is consistent?